src/Coduo/PHPHumanizer/Resources/translations/oxford.en.yml

```diff
@@ -0,0 +1,3 @@
+only_two: "%first% and %second%"
+comma_separated: "%list%, and %last%"
+comma_separated_with_limit: "{1} %list%, and 1 other|[2,Inf] %list%, and %count% others"
```

Comment:
Will do :) This one seems pretty straightforward, no caveats as in difference.xx.yaml
I could of course point out that many languages have gender-depending noun forms and in this case it affects "1 other", but this can be worked around.